packages/runtime/src/helpers/functions.ts

```diff
@@ -269,7 +269,7 @@ const traceNextServer = async (publish: string): Promise<string[]> => {
 
 export const traceNPMPackage = async (packageName: string, publish: string) => {
   try {
-    return await glob(join(dirname(require.resolve(packageName, { paths: [publish] })), '**', '*'), {
+    return await glob(join(dirname(require.resolve(packageName, { paths: [__dirname, publish] })), '**', '*'), {
```

Comment:
Are we sure this file is exectued as commonjs and not ESM? because `__dirname` is only available in commonjs.

Reply:
Okay I see that `__dirname` is also used on other locations in this file.

Reply:
It better be, because `require.resolve` is also absent from ESM.